src/spaceapi_client.rs

```diff
@@ -14,20 +14,22 @@ use std::io;
 pub struct SpaceApiClient {
   last_fetch: i64,
   webcams: Vec<String>,
+  location: Option<Location>,
```

Comment:
This line for a start.

I made it like this, so I could start the bot without the status-api. But it is no optional property of the Status data structure... so